packages/mds-policy-service/tests/policy-client.spec.ts

```diff
@@ -86,7 +86,9 @@ describe('spot check unit test policy functions with SimplePolicy', () => {
           geographies: [TEST_GEOGRAPHY_UUID1],
           days: ['mon'],
           start_time: '09:00:00',
-          end_time: '10:00:00'
+          end_time: '10:00:00',
+          vehicle_types: ['scooter'],
```

Comment:
Can we also have a test without vehicle_types?

Reply:
+1